Source/Core/DolphinQt2/Config/Graphics/EnhancementsWidget.cpp

```diff
@@ -245,12 +245,14 @@ void EnhancementsWidget::AddDescriptions()
 
   static const char* TR_UBERSHADER_DESCRIPTION =
       QT_TR_NOOP("Disabled: Ubershaders are never used. Stuttering will occur during shader "
-                 "compilation, but GPU demands are low. Recommended for low-end hardware.\n\n"
+                 "compilation, but GPU demands are low. Recommended for low-end hardware.\n"
                  "Hybrid: Ubershaders will be used to prevent stuttering during shader "
                  "compilation, but traditional shaders will be used when they will not cause "
-                 "stuttering. Balances performance and smoothness.\n\n"
+                 "stuttering. Balances performance and smoothness.\n"
                  "Exclusive: Ubershaders will always be used. Only recommended for high-end "
-                 "systems.");
+                 "systems.\n"
+                 "Skip: Does not draw objects during shader compilation. Reduces stuttering "
+                 "at the cost of potentially broken effects.");
```

Comment:
It is likely that some effects are broken temporary. It may also break them permanently. So "potentially" sounds too safe for me.